models/ticket_history/int_zendesk__field_history_pivot.sql

```diff
@@ -11,7 +11,7 @@
 }}
 
 {% if execute and flags.WHICH in ('run', 'build') -%}
-    {% set results = run_query('select distinct field_name from ' ~ var('field_history') ) %}
+    {% set results = run_query('select distinct field_name from ' ~ ref('stg_zendesk__ticket_field_history') ) %}
```

Comment:
`var('field_history')` used to point to `stg_zendesk__ticket_field_history` before the source package consolidation, but now it points to the source, so we gotta update.